packages/core/src/routes/init.ts

```diff
@@ -39,6 +40,7 @@ const createRouters = (provider: Provider) => {
   logRoutes(managementRouter);
   roleRoutes(managementRouter);
   dashboardRoutes(managementRouter);
+  customPhraseRoutes(managementRouter);
```

Comment:
Should this be coupled with our `AuthedRouter`? Or `AnonymousRouter` is good to go?

Reply:
OK, I get it now, and this `customPhraseRoutes` will only be used in admin console. Will provide other anonymous routes for usages in mainflow ui.